### ISSUE TITLE
ZeroDivisionError in Parallel Builds

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1172]
```
buildbot@adorno:~$ cat /tmp/sphinx-err-nT7HqM.log
# Sphinx version: 1.2b1
# Python version: 2.7.3
# Docutils version: 0.10 release
# Jinja2 version: 2.6
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/sphinx/cmdline.py", line 247, in main
    app.build(force_all, filenames)
  File "/usr/local/lib/python2.7/dist-packages/sphinx/application.py", line 211, in build
    self.builder.build_update()
  File "/usr/local/lib/python2.7/dist-packages/sphinx/builders/__init__.py", line 211, in build_update
    'out of date' &#37; len(to_build))
  File "/usr/local/lib/python2.7/dist-packages/sphinx/builders/__init__.py", line 267, in build
    self.write(docnames, list(updated_docnames), method)
  File "/usr/local/lib/python2.7/dist-packages/sphinx/builders/__init__.py", line 315, in write
    nproc=self.app.parallel - 1)
  File "/usr/local/lib/python2.7/dist-packages/sphinx/builders/__init__.py", line 358, in _write_parallel
    nchunks, rest = divmod(ndocs, chunksize)
ZeroDivisionError: integer division or modulo by zero

```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1172
- Originally reported by: [tychoish](https://bitbucket.org/tychoish)
- Originally created at: 2013-05-21T01:20:26.241

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2013-07-01 04:24:57+00:00_

Merged in tychoish/sphinx (https://bitbucket.org/birkenfeld/sphinx/pull-request/146)

resolve issue #1172: don't divide by zero

→ <<cset 8818b9a0c4624c409fb78a1018233e872dea7196>>

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2013-07-01 04:28:14+00:00_

Fix ZeroDivisionError in Parallel Builds. Thanks to tychoish. Closes #1172

→ <<cset ab5c7a36be65a5cdba23c038d967715a2e86231d>>
